config/prod/miac-data-model-web-ec2.yaml

```diff
@@ -0,0 +1,50 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v10.j2
+stack_name: miac-data-model-web-ec2
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project this resource will be used for
+  Project: "CSBC"
+  OwnerEmail: "milen.nikolov@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "computevpc"
+  KeyName: "scicomp"
+  AMIId: "ami-0b898040803850657"      # Amazon Linux 2 AMI 2.0.20190618 x86_64 HVM gp2
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.nano"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  #VolumeSize: "500"
```

Comment:
did you want a larger volume?

Reply:
8GB should be enough but increased to 24GB just in case.